.github/workflows/all-static-checks.yaml

```diff
@@ -88,6 +88,48 @@ jobs:
         run: |
           pip install pyyaml
           python tests/sweep_framework/framework/sweeps_workflow_verification.py
+  cmake-version:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: recursive
```

Comment:
This is necessary for the cmake build to actually go through right?

Reply:
Correct.  We pull in submodules into our build itself.  So CMake must be able to process the entire tree.
I dropped LFS, though.  That's not necessary for generating the build files.